app/components/DR/CasesStatistics.js

```diff
@@ -102,8 +107,22 @@ class CasesStatistics extends React.Component {
               styles.dateSubtitle,
               { alignSelf: 'center', textAlign: 'center' },
             ]}>
-            {this.getUpdateDate()}
+            {t('label.date_dashboard_label')}
           </Text>
+          <View style={{ alignSelf: 'center' }}>
+            <CalendarButton
+              style={{ backgroundColor: '#FFF' }}
+              date={
+                date
+                  ? moment(date, 'YYYY-MM-DD').format('DD-MM-YYYY')
+                  : moment(new Date()).format('DD-MM-YYYY')
+              }
+              onChange={date => {
+                this.getCases(moment(date, 'DD-MM-YYYY').format('YYYY-MM-DD'));
+              }}
+              minDate={'2020-07-17'}
```

Comment:
You put the minDate in an incorrect format, the format should be **DD-MM-YYYY** and the brackets are not necessary

```suggestion
              minDate='17-07-2020'
```

![image](https://user-images.githubusercontent.com/43874147/93109595-45799a00-f682-11ea-978a-75029484410b.png)

Reply:
Nop, is correct but the bracket can be deleted